go/consensus/tendermint/apps/fuzz/fuzz.go

```diff
@@ -0,0 +1,223 @@
+// +build gofuzz
```

Comment:
Any reason why this is under `tendermint/apps/fuzz`? Wouldn't it make more sense to be under `tendermint/fuzz` directly?

Reply:
I thought it makes more sense in `apps/` because the code is very specific to the transaction system we're using and it's directly related to our apps. It might make sense in the `abci/` directory alongside the muxer, but I'm not sure putting it in `tendermint/` would make sense.

Reply:
I think of this in the same way as test helpers (which live under `tendermint/tests`). The `tendermint/apps` directory contains the apps themselves.